transforms/remove-gridiron-import.js

```diff
@@ -2,6 +2,10 @@ module.exports = function (file, api, options) {
     var j = api.jscodeshift;
     var root = j(file.source);
 
+    if (!options.esprima) {
+        options.esprima = require("babel-core");
```

Comment:
@doctyper this was the solution to not being able to pass options into the `jscodeshift` cli. Maybe we could have a base class do this and have the transforms extend it, so it only would have to happen once?